todo_app/flask_config.py

```diff
@@ -11,6 +11,8 @@ def __init__(self):
         self.github_app_id = os.getenv('OAUTH_APP_ID')
         self.github_app_secret = os.getenv('OAUTH_APP_SECRET')
         self.LOGIN_DISABLED = os.getenv('LOGIN_DISABLED')
+        self.LOG_LEVEL = os.getenv('LOG_LEVEL')
```

Comment:
Your unit tests currently fail because this isn't set and so when the config tries to set it there are issues. We could add this to your `.env.test`, but possibly more sensible would be to set a default here:
```suggestion
        self.LOG_LEVEL = os.getenv('LOG_LEVEL', 'INFO')
```